inspirehep/modules/workflows/workflows/article.py

```diff
@@ -304,53 +308,47 @@
 
 
 SEND_TO_LEGACY_AND_WAIT = [
-    IF_ELSE(
+    send_robotupload(
+        marcxml_processor=hep2marc,
+        mode="replace"
+    ),
+    IF_NOT(
         is_an_update,
-        [
-            prepare_update_payload(extra_data_key="update_payload"),
-            send_robotupload(
-                marcxml_processor=hep2marc,
-                mode="correct",
```

Comment:
If we don't go the way of obtaining a recid from legacy, then it suffice to change here `correct` into `replace`.